cpp/src/parquet/bloom_filter.h

```diff
@@ -101,6 +101,12 @@ class PARQUET_EXPORT BloomFilter {
   /// @return hash result.
   virtual uint64_t Hash(const FLBA* value, uint32_t len) const = 0;
 
+  // Variant of const pointer argument to facilitate template
```

Comment:
instead of doing these, could we make const-ref parameters for the pointer types?

Reply:
It means that we wrap it in a template or "PhysicalType"?